package.json

```diff
@@ -52,7 +52,7 @@
   "dependencies": {
     "@angular-devkit/schematics": "^12.2.2",
     "@typescript-eslint/experimental-utils": "^4.31.1",
-    "eslint-etc": "^4.1.0",
+    "eslint-etc": "^4.2.4",
```

Comment:
Even without this updating, when I ran `npm i`, all the changes in `package-lock.json` made on #212, are reverted as you can see above. Thoughts?

Reply:
Oh yikes, I didn't pay attention to the lock file...
It' because I'm using npm v7, which uses a different package-lock structure.
I'm guessing you're on a lower version, using the "older" version of the package-lock

Reply:
Thanks! I've updated to npm v7 and it seems to working very well now. And it also explains the surprise I had [here](https://github.com/timdeschryver/eslint-plugin-ngrx/pull/198#discussion_r707429351)!

![now](https://user-images.githubusercontent.com/11965907/133890244-0b2ff95b-233b-446a-ae7d-ae1e639877cf.gif)

Reply:
😂